src/common/components/search-input/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { PropTypes } from 'react';
+import styles from './search.css';
+
+export default function SearchInput(props) {
+  return (
+    <label htmlFor={ props.id }>
+      { props.label }
+      <input
+        id={ props.id }
+        className={ styles.selectRepositorySearch }
+        onChange={props.onChange}
+        placeholder={ props.placeholder }
+        value={props.value}
+        type='search'
+      />
+    </label>
+  );
+}
+
+SearchInput.propTypes = {
+  id: PropTypes.string.isRequired,
+  label: PropTypes.string,
+  onChange: PropTypes.func.isRequired,
+  placeholder: PropTypes.string.isRequired,
```

Comment:
Not sure if we need to make placeholder required. At least repos search input doesn't have any on designs.